backend/services/eventService.js

```
@@ -0,0 +1,273 @@
+        await Club.findByIdAndUpdate(event.club, {
+            $pull: { createdEvents: event._id }
+        });
+
+        await User.updateMany(
+            { eventsJoined: event._id },
+            { $pull: { eventsJoined: event._id } }
+        );
+
+        await Event.deleteOne({ _id: event._id });
+    } catch (error) {
+        throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)